sdk/js-compute/index.d.ts

```diff
@@ -306,6 +306,76 @@ export declare class Env {
   get(name: string): string;
 }
 
+/**
+ * Class for accessing a [Fastly Object-store](https://developer.fastly.com/reference/api/object-store/).
+ *
+ * An object store is a persistent, globally consistent key-value store.
+ * 
+ * **Note**: Can only be used when processing requests, not during build-time initialization.
+ */
+export declare class ObjectStore {
+  /**
+   * Creates a new JavaScript ObjectStore object which interacts with the Fastly Object-store named `name`.
+   * 
+   * @param name Name of the Fastly Object-store to interact with. A name cannot be empty, contain Control characters, or be longer than 255 characters.
+   */
+  constructor(name: string);
+  /**
+   * Gets the value associated with the key `key` in the Object-store.
+   * When the key is present, a resolved Promise containing an ObjectStoreEntry will be returned which contains the associated value.
+   * When the key is absent, a resolved Promise containing null is returned.
+   * @param key The key to retrieve from within the Object-store. A key cannot:
+   * - Be any of the strings "", ".", or ".."
+   * - Start with the string ".well-known/acme-challenge/""
+   * - Contain any of the characters "#?*[]\n\r"
+   * - Be longer than 1024 characters
```

Comment:
Is this still true? The comment in object-store.cpp says that they must be less than 255 chars.

Reply:
This is still true as it is for a store key and not the store name - https://github.com/fastly/js-compute-runtime/blob/a6ab7ac951753864a922a48a8ec89ecd4f852a4f/c-dependencies/js-compute-runtime/builtins/object-store.cpp#L125